src/app/[locale]/business/products/page.tsx

```diff
@@ -0,0 +1,95 @@
+'use client';
+
+import React, { useState } from 'react';
+import { useTranslations } from 'next-intl';
+import Switch from 'react-switch';
+import { AddButton } from '@/components/shared/Forms/Buttons/Buttons';
+import { useRouter } from 'next/navigation';
+
+
+const switchProp = {
+  onColor: '#619681',
+  offHandleColor: '#5e5e5e',
+  offColor: '#b6b6b6',
+  onHandleColor: '#0a4223',
+  handleDiameter: 20,
+  height: 16,
+  width: 36,
+};
+
+function Products() {
+
+  const t = useTranslations();
+  const router = useRouter()
+  const [checked, setChecked] = useState({
+    prod_on: false,
+    pay_enable: false,
+    without_payment: false,
+  });
+
+
+  const handleChange = (check: any) => {
+    check === 'prod_on' &&
+      setChecked({ ...checked, prod_on: !checked.prod_on });
+    check === 'pay_enable' &&
+      setChecked({ ...checked, pay_enable: !checked.pay_enable });
+    check === 'without_payment' &&
+      setChecked({ ...checked, without_payment: !checked.without_payment });
+  };
+
+  const handleAddBtn = () => {
+    router.push('/add-product')
+  }
+
+
+  return (
```

Comment:
Hey @omotoshoayomikun - The Angular version of this configuration screen is actually in the business-settings/business-menu/business-menu.component.html and not the business-settings/products/products-settings/products-settings.component.html. 
There you will see the correct translated references.  Sorry for the confusion!

Reply:
I can send you the exact screenshot once the prod issue is resolved, but it's basically the same screen layout as what you have other than the toggling and button behavior.  For now, you can just copy over the corresponding translated content that exists in the /business-menu.component.html. i.e., BUSINESS.CONFIGURATION.LABELS.*  We can revisit the details of the screen later. 
Thanks!

Reply:
> I can send you the exact screenshot once the prod issue is resolved, but it's basically the same screen layout as what you have other than the toggling and button behavior. For now, you can just copy over the corresponding translated content that exists in the **/business-menu.component.html**. i.e., BUSINESS.CONFIGURATION.LABELS.* We can revisit the details of the screen later. Thanks!

Okay that's true but I think it has a pop up model so i will work on it.